lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_constants.py

```diff
@@ -0,0 +1,12 @@
+"""Module to contain constants used for Ensemble Copula Coupling."""
+
+# Specify the bounds for each phenomenon for creating the empirical
```

Comment:
I kind of feel like there needs to be a bit more detail here, either a reference and/or some information saying the meaning of these numbers. Maybe you just need the words 'end points for the distribution' or something similar. What do you think?